marketing doc updates: roles, owners, guidelines, more

parispittman opened the pull request:
some of #4516, adding team member names, more robust information for the README, addition of OWNERs (i know it's invalid comments; will take it off post discussion)

mrbobbytables commented on communication/marketing-team/social-guidelines.md:
```diff
@@ -0,0 +1,23 @@
+Bio should mention that this is powered by sig-contribex, link to the kubernetes/community repo  
+Never include someones photo, twitter handle, or other personal identifiable info without their explicit permission  
+try not to abbreviate as much as possible  
+all event tweets should have an action and/or thumbnail image    
```
These are all con'cated together.  They need either an additional line or added as a list

parispittman replied:
yep - just wanted folks to see the text first

parispittman replied:
added a numbered list

mrbobbytables commented on communication/marketing-team/social-guidelines.md:
```diff
@@ -0,0 +1,23 @@
+try not to abbreviate as much as possible  
+all event tweets should have an action and/or thumbnail image    
+
+Themes:
```
I would also add release notices such as code freeze etc to this list. 👍

parispittman replied:
added

mrbobbytables commented on communication/marketing-team/team-resources.md:
```diff
@@ -0,0 +1,21 @@
+[Community Groups List]: https://github.com/kubernetes/community/blob/master/sig-list.md
+[Values]: https://github.com/kubernetes/community/blob/master/values.md
+[principles]: https://github.com/kubernetes/community/blob/master/governance.md#principles
+[Entire community repo]: https://github.com/kubernetes/community
```
Similar to the earlier comment, these should all be relative links to their locations in the community repo.

mrbobbytables commented on communication/marketing-team/team-resources.md:
```diff
@@ -0,0 +1,21 @@
+
+[contributor site]: https://github.com/kubernetes-sigs/contributor-site
+[project board]: https://github.com/orgs/kubernetes-sigs/projects/3
+[surveys]: https://github.com/kubernetes/community/tree/master/sig-contributor-experience
```
```suggestion
[surveys]: /sig-contributor-experience/surveys
```

parispittman replied:
fixed

mrbobbytables commented on communication/marketing-team/README.md:
```diff
@@ -1,57 +1,41 @@
 the broader ecosystem
 
-# Core Marketing Team roles
+# Core Marketing Team
```
```suggestion
## Core Marketing Team
```
It's odd having an H1 below an H2 like this.

parispittman replied:
fixed

mrbobbytables commented on communication/marketing-team/README.md:
```diff
@@ -1,57 +1,41 @@
-
-# Charter
-view our charter in this [marketing folder] for more about us and how to join
+The Contributor Marketing Team is part of the community-management subproject in
```
It might be worth adding an H1 header here of `Contributor Marketing Team` or something of that sort.

parispittman replied:
fixed

mrbobbytables commented on communication/marketing-team/README.md:
```diff
@@ -1,57 +1,41 @@
+Let us know you are interested and if you have any questions.
+
+
+[meetings]: https://github.com/kubernetes/community/blob/master/sigs.yaml#L1143
```
```suggestion
[meetings]: /sig-contributor-experience/README.md#community-management
```

This will point to the wrong thing after a sigs.yaml update 😬 Probably better to link to the community-management project in the readme.

parispittman replied:
fixed

mrbobbytables commented on communication/marketing-team/README.md:
```diff
@@ -1,57 +1,41 @@
+[meetings]: https://github.com/kubernetes/community/blob/master/sigs.yaml#L1143
+[charter]:  https://git.k8s.io/community/communication/marketing-team/CHARTER.md
+[Could be you!]: (#Could-be-you!)
+[Contributor Experience]: https://git.k8s.io/community/sig-contributor-experience
```
On all the links that are self-referencing the community repo they should not have the url portion e.g.
```
[charter]:  https://git.k8s.io/community/communication/marketing-team/CHARTER.md
```
becomes:
```
[charter]: ./CHARTER.md
```

parispittman commented on communication/marketing-team/OWNERS:
```diff
@@ -0,0 +1,15 @@
+  - parispittman
+  - mbbroberg
+approvers: # all editors; lead editor should be someone in the root of
+k/community/communication but not must be - needs discussion
```
yep using this for discussion only and will make the changes after

parispittman replied:
fixed

parispittman commented on communication/marketing-team/OWNERS:
```diff
@@ -0,0 +1,15 @@
+  - chris-short
+  - onlydole
+  - parispittman
+  - mbbroberg
```
he is working on it as we speak

mbbroberg replied:
@mrbobbytables got that going: https://github.com/kubernetes/org/issues/1719

parispittman replied:
fixed

mbbroberg commented on communication/marketing-team/README.md:
```diff
@@ -1,57 +1,41 @@
-# Core Marketing Team roles
+# Core Marketing Team
 
 | Role | Handbook | Notes |
```
Given the lead/shadow model, I think shifting the `Lead` specification to the 3rd column would make the roles and titles a bit clearer, so:

parispittman replied:
I made some further modifications because I also thought it was odd originally but with this comment and @jonasrosland, I had an idea of adding an 'area' section which makes it a bit clearer with the addition of 'team' and 'lead' names. wdyt?

mbbroberg commented on communication/marketing-team/README.md:
```diff
@@ -1,57 +1,41 @@
+| Social Media Lead | TODO | @chris-short; shadow - [Could be you!] (paris to help bootstrap) |
+| Bloggers | TODO | @onlydole (liaison to sig-docs-blog team) @celanthe, @vonguard, @boluisa, [Could be you!] |
+| Designer | TODO | [Could be you!] help us with thumbnail graphics, digital assets, infographics, contributor site |
+
```
```suggestion
| Editor | TODO | Lead: @parispittman (to bootstrap); assoc editor/shadow: @mbbroberg |
| Internal Comms | TODO | Lead: @jonasrosland; shadow: @simplytunde |
| Social Media | TODO | Lead: @chris-short; shadow - [Could be you!] (paris to help bootstrap) |
| Bloggers | TODO | Lead: @onlydole (liaison to sig-docs-blog team); contributors: @celanthe, @vonguard, @boluisa, [Could be you!] |
| Designer | TODO | [Could be you!] help us with thumbnail graphics, digital assets, infographics, contributor site |

```

jonasrosland replied:
I like adding the Leads here, but perhaps that could be a header in itself?

jonasrosland replied:
Like so:
```
 | Role | Handbook | Lead | Shadow | Notes | 
```

onlydole commented:
I agree with @mbbroberg and @mrbobbytables's edits...LGTM after those are updated!

parispittman commented on communication/marketing-team/social-guidelines.md:
```diff
@@ -0,0 +1,23 @@
+Future home of social media guidelines
+
+[insert twitter handle here]
```
added

parispittman commented:
alrighty - addressed all of the comments above, added in some more formatting flair, and made some table edits to the README roles with the addition of 'area'. check it out and let me know what you think.

parispittman commented:
everyone in the new OWNERs file as weighed in. waiting for both tunde's but we can PR in further changes if they would like to see them, including swapping of roles, etc. 
same note for formatting - if we see any nits now, lets catch in a follow up PR. @mrbobbytables if thats cool with you since you had the most comments after review, undo the hold and provide your review. thanks!

mrbobbytables commented:
Looks good!
Thanks!
/lgtm
/hold cancel
